src/controller/debtor-controller.ts

```diff
@@ -0,0 +1,233 @@
+/**
+ *  SudoSOS back-end API service.
+ *  Copyright (C) 2020  Study association GEWIS
```

Comment:
Denk dat deze wel geüpdatet moet worden?

Reply:
Ja, maar laten we dat doen in een andere PR. Dit is namelijk in de hele repository fout omdat het wordt ge-enforced door de linter.